services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -67,25 +67,25 @@ as color tokens, type tokens, spacing tokens, the grid, and more.
 **Guidelines** provide direction on critical topics such as accessibility and content and ensure
 consistent user experiences that meet IBM's standards.
 
-**Assets** include re–usable resources such as components, patterns, functions, and templates. Each
-asset has code, designs, and usage guidance.
+**Assets** are reusable resources such as components, patterns, functions, and templates. Assets
+often have usage and accessibililty guidance, as well as styling specifications and code.
 
-**Design kits** are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.
-They contain components and elements as well as wireframes and guidance to aid designers.
```

Comment:
'Design Kits are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.' was something I drafted with Morgana and Scott after multiple iterations. And we had several users confusing design kits and code libraries, so this was written to add clarity. I'd like to add this back in.

Reply:
@aubrey-oneal if we say "Design kits are libraries"... doesn't that imply that you would find design kits in the libraries catalog?

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1691245/207645622-04ef2897-6854-43f4-999b-0bc9deece10a.png">

To prevent that confusion, it's my recommendation that we don't use the word "library" in describing design kits.

Why are we calling these resources "design kits" in the first place, if there are [Figma libraries](https://help.figma.com/hc/en-us/articles/360041051154-Guide-to-libraries-in-Figma), [Sketch libraries](https://www.sketch.com/docs/designing/libraries/), [Adobe XD libraries](https://www.adobe.com/products/xd/learn/get-started-xd-components-libraries.html), and [Axure widget libraries](https://www.axure.com/support/download-widget-libraries)?

Our prior decision was to index libraries, assets (that belong to libraries), and design kits (that are compatible with libraries and assets.) How can we stick to that right now?

Or is this evidence that we need to drop "design kits" all-together (not in this PR), and instead use the structure:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/1691245/207648009-88dd1522-b5ef-471a-b349-b7e6019953a6.png">

Reply:
Design kits are called libraries in the tools Sketch, Figma, Adobe XD, and Invision. Unfortunately we need to translate what we call 'libraries' for these design files into 'design kits' in the definition to add clarity that we aren't talking about code libraries. This came up in our user research, too.

Reply:
We can definitely open a new discussion on changing 'design kits' to design 'libraries. I like the symmetry with 'code libraries'!

Reply:
@aubrey-oneal I'll revert my change so the word "library" is still used to describe a design kit.

I still believe that adds confusion with the catalogs as-is. I'll open up a new issue to explore "Design libraries" and "Code libraries" classifications in the future.

Reply:
Issue for if we want to revisit this later: https://github.com/carbon-design-system/carbon-platform/issues/1550